src/components/TradeWidget/index.tsx

```diff
@@ -674,14 +674,23 @@ const TradeWidget: React.FC<TradeWidgetProps> = ({
             isPriceInverted={isPriceInverted}
           />
           <PriceSuggestions
+            // Network
```

Comment:
I like the comments. When they have to many components, if there's not another solution like break the component or simplify it signature, it's nice to at least add these comments 👌